core/src/main/scala/kafka/server/BrokerMetadataCheckpoint.scala

```diff
@@ -184,6 +185,8 @@ object BrokerMetadataCheckpoint extends Logging {
       try {
         brokerCheckpoint.read() match {
           case Some(properties) =>
+            // XXX Should we check for duplicates here
```

Comment:
Is `LogManager.directoryIds()` a better place to check for duplicates?

Reply:
I've added it there. Let me know what you think